jupyterhub/templates/scheduling/user-scheduler/configmap.yaml

```
@@ -6,83 +6,6 @@ metadata:
   labels:
     {{- include "jupyterhub.labels" . | nindent 4 }}
 data:
-  policy.cfg: |
-    {
-      "kind": "Policy",
-      "apiVersion": "v1",
-      "predicates": [
-        { "name": "PodFitsResources" },
-        { "name": "HostName" },
-        { "name": "PodFitsHostPorts" },
-        { "name": "MatchNodeSelector" },
-        { "name": "NoDiskConflict" },
-        { "name": "PodToleratesNodeTaints" },
-        { "name": "MaxEBSVolumeCount" },
-        { "name": "MaxGCEPDVolumeCount" },
-        { "name": "MaxAzureDiskVolumeCount" },
-        { "name": "CheckVolumeBinding" },
-        { "name": "NoVolumeZoneConflict" },
-        { "name": "MatchInterPodAffinity" }
-      ],
-      "priorities": [
-        { "name": "NodePreferAvoidPodsPriority",  "weight": 161051 },
-        { "name": "NodeAffinityPriority",         "weight": 14641 },
-        { "name": "InterPodAffinityPriority",     "weight": 1331 },
-        { "name": "MostRequestedPriority",        "weight": 121 },
-        { "name": "ImageLocalityPriority",        "weight": 11}
-      ],
-      "hardPodAffinitySymmetricWeight" : 100,
-      "alwaysCheckAllPredicates" : false
-    }
+  {{ $defaultPolicy := .Files.Get "files/userscheduler-defaultpolicy.yaml " | fromYaml}}
```

- Chomp left to not cause a blank line after data:
- remove a trailing white space in the path
- nitpick helm formatting by adding a space

```suggestion
  {{- $defaultPolicy := .Files.Get "files/userscheduler-defaultpolicy.yaml" | fromYaml }}
```